net/release-notes/release-notes-2021/aspose-pdf-for-net-21-7-release-notes/_index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Aspose.PDF for .NET 21.7 Release Notes
+linktitle: Aspose.PDF for .NET 21.7 
+type: docs
+weight: 70
+url: /net/aspose-pdf-for-net-21-7-release-notes/
+lastmod: "2021-07-13"
+description: This page contains new Aspose.PDF for .NET features, enhancement, and bug fixes in 2021, version 21.7. 
+sitemap:
+    changefreq: "weekly"
+    priority: 0.7
+---
+
+{{% alert color="primary" %}}
+
+This page contains release notes information for Aspose.PDF for .NET 21.7.
+
+{{% /alert %}} 
+
+## Improvements and Changes
+
+|**Key**|**Summary**|**Category**|
+| :- | :- | :- |
+|PDFNET-49595 Add memory optimized stream to load huge files using Stream based constructor|Feature|
```

Comment:
Advanced support of Streams / Advanced handling huge documents

Reply:
Advanced streams support to handle huge documents?